content/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Allow_List.adoc

```diff
@@ -0,0 +1,114 @@
+
+This scenario works best when:
+
+* You are an organization that only uses a small set of specific authenticators
+* You want to prevent users from registering authenticators like copyable passkeys
+* You are required to limit registrations to high assurance devices
+
+=== User experience
+
+Figure 1 demonstrates a sample user experience for a registration process that implements an allow list
+
+image::../images/allowlist1.jpg[]
+**Figure 1**
+
+In this flow the happy path comes from a user who immediately opts into sending attestation data, and ensuring that they are using an authenticator that is allowed in your environment. 
+
+It’s worth noting that in this experience most of the logic/decision points are left up to the application, with little intervention from the user, aside from creating the initial registration, and reacting to any request rejections.
+
+Here are some considerations for this flow:
+
+* Ensure that your users know to allow for attestation data to be sent during the registration ceremony, this can be done by modals, tipboxes, or instructions on your client application
+* Ensure that users know which of their authenticators to use for the application
+* Ensure error messages accurately reflect the issue at hand so that users have a better chance to self remediate
+
+=== Prerequisites
+Before you attempt to follow this implementation guidance, ensure that you have added the MDS to your application. Guidance for adding MDS to your application can be found on our page, link:​​/WebAuthn/Concepts/FIDO_Metadata_Service_(MDS).html[FIDO Metadata Service (MDS)]
+
+Our examples also leverage Yubico’s java-webauthn-server; a server side library for Java that provides features to help you implement a WebAuthn relying party. Some of the methods used below will come directly from this library.
+
+link:https://github.com/Yubico/java-webauthn-server[Link to the java-webauthn-server]
+
+=== Implementation guidance
+
+==== Allow only trusted attestation
+
+One immediate modification will be needed from the code introduced in the MDS guidance linked above. In this flow you should opt to **only** accept registrations from trusted attestation sources. This means that any registration sent to your application should include an attestation statement. If a user opts not to send an attestation statement, then their registration should be rejected. A user request may also be rejected if the authenticator they are using is not included in the MDS.
```

Comment:
Consider changing 'should' to 'must'. e.g. "must include an attestation statement."
Also consider mentioning that anonymous attestation statements are also untrusted and will be rejected.